dist/api_data.py

```diff
@@ -814,6 +814,15 @@ def __ge__(self, other):
         using the \c statistics_log configuration. See @ref statistics for more information''',
         type='list',
         choices=['all', 'cache_walk', 'fast', 'none', 'clear', 'tree_walk']),
+    Config('strategy', '', r'''
```

Comment:
The name `strategy` feels overly generic to me. I agree with the intention, but maybe it could be something like `heuristic_controls` (a bit wordy, but I'm not much good at naming)

Reply:
Thanks. Changed.